package.json

```
@@ -8,7 +8,8 @@
   ],
   "scripts": {
     "test": "jest --testTimeout=60000 --forceExit",
-    "build": "node ./src/build.js"
+    "build": "node ./src/build.js",
+    "postinstall": "node ./templates/readmeTemplate.mjs"
```

This will fix the tests failing:
```suggestion
    "postinstall": "cd templates && node ./readmeTemplate.mjs"
```